Mathlib/ModelTheory/Arithmetic/Presburger/Semilinear/Defs.lean

```diff
@@ -44,9 +44,8 @@ of sets in form `{ x | ∃ y, p x y }`.
 
 universe u v
 
-variable {α : Type u} {β : Type v} [AddCommMonoid α] [AddCommMonoid β]
-  {F : Type*} [FunLike F α β] [AddMonoidHomClass F α β]
-  {ι κ : Type*} {a : α} {s s₁ s₂ : Set α}
+variable {α β ι κ F : Type*} [AddCommMonoid α] [AddCommMonoid β]
+  [FunLike F α β] [AddMonoidHomClass F α β] {a : α} {s s₁ s₂ : Set α}
```

Comment:
```suggestion
variable {M N ι κ F : Type*} [AddCommMonoid M] [AddCommMonoid N]
  [FunLike F M N] [AddMonoidHomClass F M N] {a : M} {s s₁ s₂ : Set M}
```
since they are monoids and not arbitrary types